safe_transaction_service/history/helpers.py

```diff
@@ -106,7 +106,7 @@ def calculate_hash(
                     {"name": "chainId", "type": "uint256"},
                 ],
                 "Delegate": [
-                    {"name": "delegateAddress", "type": "bytes32"},
+                    {"name": "delegateAddress", "type": "address"},
```

Comment:
This will be a breaking change, no?

Reply:
Kind of yes. I've mentioned this in the PR description.

Reply:
We should make the same change here: https://github.com/safe-global/safe-eth-py/blob/d780c51039ebcc2593f80dbc4e8f1140aadfe019/gnosis/safe/api/transaction_service_api/transaction_service_messages.py#L18

Reply:
> We should make the same change here: https://github.com/safe-global/safe-eth-py/blob/d780c51039ebcc2593f80dbc4e8f1140aadfe019/gnosis/safe/api/transaction_service_api/transaction_service_messages.py#L18

Added it here. https://github.com/safe-global/safe-eth-py/pull/1176

Thanks @falvaradorodriguez !